packages/components/psammead-navigation/src/index.jsx

```diff
@@ -157,45 +214,73 @@ const StyledNav = styled.nav`
   }
 `;
 
-const Navigation = ({ children, dir }) => (
-  <StyledNav role="navigation" dir={dir}>
-    <NavWrapper>{children}</NavWrapper>
-  </StyledNav>
-);
+const SwipeableNav = styled.div`
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    white-space: nowrap;
+    overflow-x: scroll;
+    scroll-behavior: smooth;
+    -webkit-overflow-scrolling: touch;
 
-Navigation.propTypes = {
-  children: node.isRequired,
-  dir: oneOf(['ltr', 'rtl']),
-};
+    /* Hide scrollbar */
+    scrollbar-width: none;
+    -ms-overflow-style: none;
+    &::-webkit-scrollbar {
+      display: none;
+    }
 
-Navigation.defaultProps = { dir: 'ltr' };
+    &:before {
+      content: ' ';
+      height: 100%;
+      width: 3rem;
+      position: absolute;
+      ${({ dir }) => css`
+        ${dir === 'ltr' ? 'right' : 'left'}: 0;
+      `}
+      bottom: 0;
+      z-index: 3;
+      overflow: hidden;
+      pointer-events: none; /* ignore clicks */
+      background: linear-gradient(
+        ${({ dir }) => (dir === 'ltr' ? 'to right' : 'to left')},
+        ${C_POSTBOX_TRANSPARENT} 0%,
+        ${C_POSTBOX_OPAQUE} 100%
+      );
+    }
+  }
+`;
 
-NavigationUl.propTypes = {
-  children: node.isRequired,
-};
+const Navigation = ({ children, dir }) => {
+  const [width, setWidth] = useState(window.innerWidth);
```

Comment:
Hmm, just thinking, how does this work on the server? Is innerWidth undefined at this point?

Reply:
I have added a condition so if it is executed on the server-side, the value of the width will be undefined.